server/src/main/java/com/tencent/rss/server/MultiStorageManager.java

```diff
@@ -140,6 +141,12 @@ public MultiStorageManager(ShuffleServerConf conf, String shuffleServerId) {
       throw new IllegalArgumentException("The value of maxShuffleSize must be positive");
     }
 
+    long pendingEventTimeoutS = conf.get(ShuffleServerConf.PENDING_EVENT_TIMEOUT_SEC);
+    if (pendingEventTimeoutS <= 0 || pendingEventTimeoutS > 1000 * shuffleExpiredTimeoutMs) {
```

Comment:
why do we compare shuffleExpiredTimeoutMs and pendingEventTimeoutS?They are irrelevant.